### DISCUSSION TITLE
[knitr] Demos: Source and output of demos | /knitr/demo/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#846] [Comments]
https://yihui.org/knitr/demo/

[a comment by an automated account is not shown]

**Reply by cwilligv** [reply]
Hey, it's been a while since you posted your question so I guess you've already figured it out. Anyways I wanted to comment that If you use the RStudio IDE, you can press the F7 key or click the menu Edit -> Check Spelling to spell-check an Rmd document.
Cheers.

[a comment by an automated account is not shown]
